src/AzureRenderHub/AzureRenderHub.WebApp/Views/Storage/View/Deploying.cshtml

```diff
@@ -33,55 +31,34 @@
-                {
-                    <h3>Deleting the storage @Model.Name</h3>
-                    <p></p>
-                }
                 else if (Model.State == StorageState.Failed)
                 {
                     <h3>Deployment of storage @Model.Name Failed</h3>
                     <p></p>
                 }
             </div>
         </div>
-        @if (Model.State == StorageState.Deleting)
-        {
-            <div class="section-wrapper">
-                <div class="section-bar"></div>
-                <div class="section-content">
-                    <div class="form-element half">
-                        <label asp-for="ResourceGroupName">Resource Group</label>
-                        <a href="@Model.ResourceGroupUrl" target="_blank">@Model.ResourceGroupName <i class="fa fa-external-link"></i></a>
-                    </div>
+        <div class="section-wrapper">
+            <div class="section-bar"></div>
+            <div class="section-content">
+                <div class="form-element half">
+                    <label asp-for="DeploymentName">Deployment Name</label>
+                    <a href="@Model.DeploymentUrl" target="_blank">@Model.DeploymentName <i class="fa fa-external-link"></i></a>
                 </div>
-            </div>
-        }
-
-        @if (Model.State != StorageState.Deleting)
-        {
-            <div class="section-wrapper">
-                <div class="section-bar"></div>
-                <div class="section-content">
-                    <div class="form-element half">
-                        <label asp-for="DeploymentName">Deployment Name</label>
-                        <a href="@Model.DeploymentUrl" target="_blank">@Model.DeploymentName <i class="fa fa-external-link"></i></a>
-                    </div>
-                    <div class="form-element half">
-                        <label asp-for="ResourceGroupName">Resource Group</label>
-                        <a href="@Model.ResourceGroupUrl" target="_blank">@Model.ResourceGroupName <i class="fa fa-external-link"></i></a>
-                    </div>
-                    <div class="form-element">
-                        <label asp-for="DeploymentState">Deployment State</label>
-                        <input asp-for="DeploymentState" readonly="readonly" />
-                    </div>
-                    <div class="form-element">
-                        <label asp-for="State">Storage State</label>
-                        <input asp-for="State" readonly="readonly" />
-                    </div>
+                <div class="form-element half">
+                    <label asp-for="ResourceGroupName">Resource Group</label>
+                    <a href="@Model.ResourceGroupUrl" target="_blank">@Model.ResourceGroupName <i class="fa fa-external-link"></i></a>
+                </div>
+                <div class="form-element">
+                    <label asp-for="DeploymentState">Deployment State</label>
```

Comment:
More nitting: "s" for "State". would be good to standardize on lower case for non product name things (unless it's the first word). We're doing this most other places in the portal.